backend/entities/Project.cs

```diff
@@ -111,6 +111,23 @@ public class Project : BaseEntity
         public string AgencyResponseNote { get; set; }
         #endregion
 
+        #region SPL
+        /// <summary>
+        /// get/set - Note summerizing offers received.
+        /// </summary>
+        public string OffersNote { get; set; }
+
+        /// <summary>
+        /// get/set - Text field describing project purchaser.
+        /// </summary>
+        public string Purchaser { get; set; }
+
+        /// <summary>
+        /// get/set - Whether or not the contract for the project is conditional
+        /// </summary>
+        public bool? IsContractConditional { get; set; }
```

Comment:
Where you planning to move this to the agency response table later?

Reply:
This one no. To me the agency response table is for per-agency data. I'm currently under the impression that only one contract can be accepted at a time, so there wouldn't be a benefit to adding a column for every row.